### ISSUE TITLE
Repo Info Worker fails, no commit data

### ISSUE THREAD

**Issue by jberkus** [chaoss/augur#1430]
**Description:**

Currently our database is not receiving any data on commits.  The repo_info_worker is erroring out.  I believe these two are related?

**How to reproduce:**
1. Run 20.2 against a set of github repos
2. Check the commits table.  It's empty
3. Check the repo_info_worker log.  See errors.
4. 
**Log files**

```
2021-08-16 17:12:03,906 [PID: 2276141] workers.repo_info_worker.50952 [register_task_failure() in worker_base.py:L438] [ERROR]: Worker ran into an error for task: {'job_type': 'MAINTAIN', 'models': ['repo_info'], 'display_name': 'repo_info model for url: https://github.com/konveyor/oadp-capstone.git', 'given': {'github_url': 'https://github.com/konveyor/oadp-capstone.git'}}
2021-08-16 17:12:03,906 [PID: 2276141] workers.repo_info_worker.50952 [register_task_failure() in worker_base.py:L439] [ERROR]: Worker was processing this task for -0.47501444816589355 seconds.
2021-08-16 17:12:03,906 [PID: 2276141] workers.repo_info_worker.50952 [register_task_failure() in worker_base.py:L443] [ERROR]: Printing traceback...
2021-08-16 17:12:03,906 [PID: 2276141] workers.repo_info_worker.50952 [register_task_failure() in worker_base.py:L444] [ERROR]: name 'repo_id' is not defined
2021-08-16 17:12:03,906 [PID: 2276141] workers.repo_info_worker.50952 [register_task_failure() in worker_base.py:L446] [ERROR]: Traceback (most recent call last):
  File "/home/augur/.virtualenvs/augur_env/lib64/python3.8/site-packages/workers/worker_base.py", line 180, in collect
    model_method(message, repo_id)
  File "/home/augur/.virtualenvs/augur_env/lib64/python3.8/site-packages/workers/repo_info_worker/repo_info_worker.py", line 203, in repo_info_model
    forked = self.is_forked(owner, repo)
  File "/home/augur/.virtualenvs/augur_env/lib64/python3.8/site-packages/workers/repo_info_worker/repo_info_worker.py", line 252, in is_forked
    data = self.get_repo_data(url, r)
  File "/home/augur/.virtualenvs/augur_env/lib64/python3.8/site-packages/workers/repo_info_worker/repo_info_worker.py", line 302, in get_repo_data
    self.register_task_failure(self.task, repo_id, "Failed to hit endpoint: {}".format(url))
NameError: name 'repo_id' is not defined

2021-08-16 17:12:03,939 [PID: 2276141] workers.repo_info_worker.50952 [register_task_failure() in worker_base.py:L490] [ERROR]: Recorded job error in the history table for: {'job_type': 'MAINTAIN', 'models': ['repo_info'], 'display_name': 'repo_info model for url: https://github.com/konveyor/oadp-capstone.git', 'given': {'github_url': 'https://github.com/konveyor/oadp-capstone.git'}, 'worker_id': 'workers.repo_info_worker.50952'}
```

**Software versions:**
 - Augur: 20.2
 - OS: CentOS Stream

**Comment by jberkus** [comment]
Possibly related to: #1417

**Comment by jberkus** [comment]
This was resolved with a re-install of 20.2.
